workshop/exercise-3/README.md

```diff
@@ -1,68 +1,80 @@
-
-	```
-	kubectl apply --filename https://github.com/knative/serving/releases/download/v0.3.0/istio-crds.yaml
-	```
-
-2. Install Istio:
-
-	```
-	kubectl apply --filename https://github.com/knative/serving/releases/download/v0.3.0/istio.yaml
-	```
-3. Label the default namespace with `istio-injection=enabled`:
-
-	```
-	kubectl label namespace default istio-injection=enabled
-	```
-
-4. Monitor the Istio components until all of the components show a `STATUS` of
-    `Running` or `Completed`:
-
-    ```
-    kubectl get pods --namespace istio-system --watch
-    ```
+Knative is currently built on top of both Kubernetes and Istio.
+If you want to learn more about Kubernetes or Istio, you can check out the
+labs [Kube101](https://github.com/IBM/kube101/tree/master/workshop) and
+[Istio101](https://github.com/IBM/istio101/tree/master/workshop).
+Luckily, when you install Knative on IKS, it will install Istio for you
+automatically.
 
 ### Install Knative
 
-After installing Istio, you can install Knative. For this lab, we will install all available Knative components.
-
-1. Install Knative Serving & Build components:
-
-    ```
-    kubectl apply --filename https://github.com/knative/serving/releases/download/v0.3.0/serving.yaml \
-    --filename https://github.com/knative/build/releases/download/v0.3.0/release.yaml
-    ```
-
-2. Monitor the Knative components until all of the components show a `STATUS` of `Running` or `Completed`:
+1. To install Knative, first make sure you have the latest Kubernetes
+   plugin:
 
-	Commands:
     ```
-    kubectl get pods --namespace knative-serving
-    kubectl get pods --namespace knative-build
+    ic plugin update
     ```
-    Example Ouput:
 
-    ```
-    NAME                          READY   STATUS    RESTARTS   AGE
-    activator-df78cb6f9-jpvs7     2/2     Running   0          38s
-    activator-df78cb6f9-nhzhf     2/2     Running   0          37s
-    activator-df78cb6f9-qjg8w     2/2     Running   0          37s
-    autoscaler-6fccb66768-m4f2q   2/2     Running   0          37s
-    controller-56cf5965f5-8pwcg   1/1     Running   0          35s
-    webhook-5dcbf967cd-lxzmk      1/1     Running   0          35s
-    ```
+2. Next ask for Knative to be installed:
 
     ```
-    NAME                                READY   STATUS    RESTARTS   AGE
-    build-controller-747b8fd966-4n8b2   1/1     Running   0          47s
-    build-webhook-6dc78d8f6d-gsm4k      1/1     Running   0          47s
-    ```
+	ic ks cluster-addon-enable knative --cluster <your_cluster_name>
+	```
 
+3. The install process may take a minute or two. To know when it's done you
+   can run two commands - first see if the Istio and Knative namespaces
+   are there:
+
+   ```
+   kubectl get ns
```

Comment:
for most commands, i'm trying to use extended version so it's clear what they're doing. Can we use namespace instead of ns.